ansible/README.md

```diff
@@ -16,16 +17,31 @@
 ## Getting things done
 
 Most of your work will probably include editing `inventory.yml`, followed by
-running one (or multiple) of below playbooks. If you're adding a new host,
-limiting ansible to just running on that host is probably quicker:
+running one (or multiple) of below playbooks.
+
+See the [manual setup instructions](../doc/non-ansible-configuration-notes.md)
+for how to prepare both the control and target machines to run the commands
+below. To ensure that the secrets are in place and test the connection to a
+host use:
+
+```console
+$ ansible test-digitalocean-debian8-x64-1 -m ping -vvvv
+Or, for Windows hosts:
```

Comment:
Maybe add a `#` so this appears as a comment

Reply:
This highlights as `console`, meaning the prompt starts a command line. If I only add a `#` it looks bad, so I added two. Could also have added the prompt `$ #`, let me know if you prefer this. Any of these looks better than dividing this into two blocks I think.